modules/virt-creating-linux-bridge-nad-cli.adoc

```diff
@@ -47,7 +47,7 @@ spec:
 <3> The name for the configuration. It is recommended to match the configuration name to the `name` value of the network attachment definition.
 <4> The actual name of the Container Network Interface (CNI) plugin that provides the network for this network attachment definition. Do not change this field unless you want to use a different CNI.
 <5> The name of the Linux bridge configured on the node.
-<6> Optional: Flag to enable MAC spoof check. When set to `true`, you cannot change the MAC address of the pod or guest interface. This attribute provides security against a MAC spoofing attack by allowing only a single MAC address to exit the pod.
+<6> Optional: A flag to enable MAC spoof check. When set to `true`, you cannot change the MAC address of the pod or guest interface. This attribute provides security against a MAC spoofing attack by allowing only a single MAC address to exit the pod.
```

Comment:
```suggestion
<6> Optional: A flag to enable the MAC spoof check. When set to `true`, you cannot change the MAC address of the pod or guest interface. This attribute provides security against a MAC spoofing attack because only a single MAC address can exit the pod.
```

Reply:
Hi Agil, thank you for the review! You're right about the definite article, but I would not change the final clause. The wording you suggested could imply that only a single MAC address being able to exit the pod is a general state, rather than something you config.

Thanks again,
J.

Reply:
OK, how about a lil bit more tweaking to avoid passive voice?
s/ "A flag to enable MAC spoof check. If set to `true`, you cannot change the MAC address of the pod or guest interface. This attribute only lets a single MAC address to exit the pod, which provides security against a MAC spoofing attack."

Reply:
Well, `This attribute provides security against a MAC spoofing attack by allowing only a single MAC address to exit the pod.` does not really contain a passive voice voice, AFAICT, but you're right that reordering the clauses makes it easier to read and comprehend. Thanks!